docs/README.md

```diff
@@ -1,117 +1,67 @@
+## Hyperledger Fabric Documentation
+The Hyperledger [fabric](https://github.com/hyperledger/fabric) is an implementation of blockchain technology, that has been collaboratively developed under the Linux Foundation's [Hyperledger Project](http://hyperledger.org). It leverages familiar and proven technologies, and offers a modular architecture that allows pluggable implementations of various function including membership services, consensus, and smart contracts (chaincode) execution. It features powerful container technology to host any mainstream language for smart contracts development.
+
+Below, you'll find the following sections:
+- [Getting started](#getting-started)
+- [Quickstart](#quickstart-documentation)
+- Developer guides
+  - [Fabric developer guide](#fabric-developer-guide)
+  - [Chaincode developer guide](#chaincode-developer-guide)
+  - [API developer guide](#api-developer-guide)
+- [Operations guide](#operations-guide)
+- [Security administration guides](#security-administration-guides)
 
 ## Getting started
 
-Welcome to the Linux Foundation Hyperledger Project documentation README. This page contains: 
+If you are new to the project, you can begin by reviewing the following links. If you'd prefer to dive right in, see the [Quickstart](#quickstart-documentation) section, below.
+- [Whitepaper WG](https://github.com/hyperledger/hyperledger/wiki/Whitepaper-WG): where the community is developing a whitepaper to explain the motivation and goals for the project
```

Comment:
Period usage seems to differ between link descriptions. Some contain a period, others do not. Also, some links are followed by ': ' where others are followed by ' - '. Otherwise LGTM